modules/localization/msf/common/io/velodyne_utility.cc

```diff
@@ -151,7 +151,7 @@ void LoadPosesAndStds(const std::string& file_path,
       timestamps->push_back(timestamp);
 
       Eigen::Vector3d std;
-      std << std_x, std_y, std_z;
+      AINFO << std_x, std_y, std_z;
       stds->push_back(std);
```

Comment:
Please make sure this logic is correct. The original implementation sets values for a vector. 

Also please make sure that AINFO is only used for initialization/check purpose. Other runtime info should be used together with ADEBUG.

Reply:
Fixed.